setup.py

```diff
@@ -94,7 +94,7 @@ def find_version(*file_paths):
         'sqlparse==0.2.4',
         'pymongo>=3.2.0',
         'django>=2.0',
-        'dataclasses>=0.1',
+        'dataclasses==0.6',
```

Comment:
You should delete this line, dataclasses is included in python already.

Reply:
So what to do in python 3.6? I don't want to answer your stupid questions, please don't ask them again.